view/popup.html

```diff
@@ -162,6 +162,7 @@
                 <a class="button" v-on:click="showEdgeBugWarning()" v-if="isEdge()">{{ i18n.download_backup }}</a>
                 <a download="authenticator.json" v-bind:href="exportFile" class="button" target="_blank" v-else>{{ i18n.download_backup }}</a>
                 <a download="authenticator.json" v-bind:href="exportEncryptedFile" v-if="encryption.getEncryptionStatus()" class="button" target="_blank">{{ i18n.download_enc_backup }}</a>
+                <a download="authenticator.txt" v-bind:href="exportOneLineOtpAuthFile" class="button" target="_blank">{{ i18n.download_otp_backup }}</a>
```

Comment:
Additionally, we should probably make a learn more link and put all the projects that use this format in a wiki page. E.g. "You can import backups from some other applications. [Learn more](https://github.com/Authenticator-Extension/Authenticator/wiki/Standard-OTP-Backup-Format)"

Reply:
I agree the new button may confuse the user. However, as I mentioned above, I haven't found a good solution to contain steam and battle.net entries on otpauth url.

Reply:
I think we could either
- Use our own convention E.g.: `otpauth://steam/`
- Just force people who use those steam and blizz secrets to use encrypted backups. All we need to do is make a popup if those types exist then. I doubt too many people use steam and bliz since you need a rooted phone to get the secret. 
- Secretly use the older format for people with those secrets and document on the learn more wiki page

Reply:
Wiki page is ready

Reply:
fixed